### PR TITLE
Fix joining issue

### PR DESCRIPTION
**Pull request by theVDude**
ircnode bot sent join command too quickly and wouldn't join some servers, but now it can.

### CODE REVIEW

File: client.js [hunk 1]
```diff
@@ -279,13 +280,15 @@ irc._socket = net.connect(irc.config.port, irc.config.address, function () {
...
+          for (var i = 0, l = chan.length; i < l; i += 1) {
+            irc.join(chan[i]);
+          }
+        } else {
+          irc.join(chan);
         }
-      } else {
-        irc.join(chan);
-      }
+      }, waitTime*1000);
```
Comment (totallymike) [inline client.js]: Make this just waitTime, and we'll put in the docs that it's milliseconds.

**Comment by totallymike** [comment]
Would you mind linting this?

Grab the jshintrc file out of my dotfiles repo, put it in ~/.jshintrc, then

``` sh

sudo npm install -g jshint
jshint client.js

```

**Comment by theVDude** [comment]
jshint cleared, still a bit unsure of js syntax

**Comment by totallymike** [comment]
Don't feel bad being less than sure about the syntax.  The syntax is so fast and loose that you can get away with a lot.  That's why we have linters ;)  jshint is pretty good for catching funky things one may have forgotten, and enforcing consistent style rules.

**Pull request merged**